### PR TITLE
Remove bintray leftovers / Fix snapshot version badge

### PR DESCRIPTION
**Pull request by mkurz**
See #3792

Page is outdated: https://doc.akka.io/docs/akka-http/current/contributing.html

### CODE REVIEW

File: docs/src/main/paradox/contributing.md [hunk 1]
```diff
@@ -38,10 +38,10 @@ Gradle
...
-        url  "https://dl.bintray.com/akka/snapshots"
+        url  "https://oss.sonatype.org/content/repositories/snapshots"
       }
     }
     ```
 
-[bintray-badge]:  https://api.bintray.com/packages/akka/snapshots/akka-http/images/download.svg
-[bintray]:        https://bintray.com/akka/snapshots/akka-http/_latestVersion
\ No newline at end of file
+[SonatypeSnapshots-badge]:  https://img.shields.io/nexus/s/https/oss.sonatype.org/com.typesafe.akka/akka-http-core_2.13.svg
```
Comment (raboof) [inline docs/src/main/paradox/contributing.md:46]: Nice!

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
OK TO TEST

**Review by ignasi35 — APPROVED** [review]
Thanks @mkurz

**Comment by jrudolph** [comment]
Great, thanks!

**Pull request merged**